themes/default/content/docs/intro/languages/javascript.md

```diff
@@ -12,7 +12,7 @@ aliases: ["/docs/reference/javascript/"]
 
 <img src="/logos/tech/logo-nodejs.png" align="right" width="150" style="padding:8px; margin-top: -64px">
 
-Pulumi supports JavaScript programs running on Node.js using any of the [Current, Active and Maintenance LTS versions](https://nodejs.org/en/about/releases/).
+Pulumi supports writing your infrastructure as code in any JavaScript language running on Node.js using any of the [Current, Active and Maintenance LTS versions](https://nodejs.org/en/about/releases/).
```

Comment:
(these _are_ proper nouns, as they are used like product names in the linked page)

Reply:
we could prolly drop the casing on our page though

Reply:
I don't understand what "any JavaScript language" means. There is only one JavaScript language, even if there are different reference implementations.

Reply:
yeh I was trying to succinctly cover JavaScript/TypeScript/anything else that transpiled.